doc/source/conf.py

```diff
@@ -22,13 +22,19 @@
 import os
 import pathlib
 import re
+from datetime import datetime
 
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'Apache Arrow Flight SQL adapter for PostgreSQL'
-copyright = '2022-2023, Apache Arrow Developers'
-author = 'Apache Arrow Developers'
+copyright = f'2022-{datetime.now().year}, Apache Software Foundation'
+trademark = [
+    "Apache Arrow Flight, Arrow Flight, Arrow Flight SQL, Apache, the Apache feather logo, and the Apache Arrow project logo",
+    "are either registered trademarks or trademarks of The Apache Software Foundation in the United States and other countries."
+]
```

Comment:
> Can we use `copyright` instead of introducing `trademark` like [apache/arrow#39505](https://github.com/apache/arrow/pull/39505) ? I think that using `copyright` is simpler.

That would be elegant, I tried the following but for some reason it doesn't split the iterable into multiple lines and the Sphinx version looks squeezed (even though the [docs state from Sphinx v7.1 onwards it should work](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright)):
![image](https://github.com/apache/arrow-flight-sql-postgresql/assets/10134699/96502037-2907-498a-88ce-410141a1b01b)


```suggestion
copyright = (f'2022-{datetime.now().year}, Apache Software Foundation\n',
             "Apache Arrow Flight, Arrow Flight, Arrow Flight SQL, Apache, the Apache feather logo, and the Apache Arrow project logo",
    "are either registered trademarks or trademarks of The Apache Software Foundation in the United States and other countries."
             )
trademark = []
```